test/Microsoft.AspNetCore.Razor.Evolution.Test/IntegrationTests/CodeGenerationIntegrationTest.cs

```diff
@@ -1626,6 +1612,10 @@ private void RunRuntimeTagHelpersTest(IEnumerable<TagHelperDescriptor> descripto
                     builder.AddTagHelpers(descriptors);
                 });
             var document = CreateCodeDocument();
+            if (tagHelperPrefix != null)
+            {
+                document.SetTagHelperPrefix(tagHelperPrefix);
+            }
```

Comment:
Agree, we didn't do this in the past because the logic to understand `@tagHelperPrefix` wasn't in the core of Razor. Aka, this is just ported code smell.